cypress/integration/singlePetPage_spec.js

```diff
@@ -1,27 +1,55 @@
 describe('Pet Applications Component', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: 'POST',
+        url: 'https://not-fur-gotten-be.herokuapp.com/graphql',
+        headers: {
+          'x-gql-operation-name': 'getPetById',
+        }
+      },
+      {
+        fixture: 'singlePetPage.json'
+      }
+    ).as('getSinglePet');
+  });
+
   it('Should display the nav bar, title, and applications for a pet', () => {
     cy.visit('http://localhost:3000/pet/1/')
+    cy.wait('@getSinglePet')
       .get('h1')
       .contains('Not Fur-gotten')
       .get('p')
       .contains('Back to Home Page')
       .get('.about-owner-nav')
-      .contains('Read More About the Owner >>')
-      .get('.single-pet-name')
-      .contains('Stewart')
+      .contains('Read More About the Owner »')
+      .get('.single-pet-owner-name')
+      .contains('Clifford')
```

Comment:
Fixed to reflect the stubbed data rather than whatever was initially hard-coded.